### ISSUE TITLE
Editor "userstate" for collaborative sessions

### ISSUE THREAD

**Issue by EwanGreen4** [yuxshao/ptcollab#53]
When following a user and then returning back to non-follow mode, settings like playhead follow mode, h/v zoom, selected unit, and more are not set back to what they were initially & are instead kept from the user being watched. Potentially make it so this isn't the case

**Comment by yuxshao** [comment]
yeah I'm not super sure if this is the right thing to do. There is a userstate that's saved already so coding-wise I don't think it's hard, but often I do want to end up at the place where I'm following vs jumping back to where I was before. If others feel the same maybe can add a setting

**Comment by EwanGreen4** [comment]
Setting is a good idea, but I think that more often than not the user would rather keep their existing workflow than switch back to their preferred zoom and stuff
